.github/workflows/osx-test-with-brew.yml

```diff
@@ -15,14 +15,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Install snap7
-        run: brew install snap7 python@${{ matrix.python-version }}
+      - name: Unlink existing Python
+        run: brew unlink python@${{ matrix.python-version }} || true
```

Comment:
i don't think this actually required, since we explicitly run the matrix.python-version binary belwo